src/sentry/static/sentry/app/components/assistant/guideDrawer.jsx

```diff
@@ -15,14 +15,27 @@ export default class GuideDrawer extends React.Component {
     step: PropTypes.number.isRequired,
     onFinish: PropTypes.func.isRequired,
     onDismiss: PropTypes.func.isRequired,
+    orgSlug: PropTypes.string,
   };
 
   handleUseful = useful => {
     markUseful(this.props.guide.id, useful);
     this.props.onFinish();
   };
 
+  interpolate(template, variables) {
+    let regex = /\${[^{]+}/g;
+    return template.replace(regex, match => {
+      const path = match.slice(2, -1).trim();
```

Comment:
Use a capture group instead of slice + trim

Reply:
fixed